changelog/8498

```diff
@@ -0,0 +1,7 @@
+Bugfix:  PDF display issue - Update CSP object-src policy
```

Comment:
Could you please give the changelog item a proper title though? Something like `bugfix-oc10-pdf-display`.

Reply:
Done in https://github.com/owncloud/web/pull/8498/commits/7b27c7c34af49229ba179ae747d26ae1510d5333 @JammingBen